feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/domain/entity/AuthenticationType.kt

```diff
@@ -3,14 +3,13 @@ package app.k9mail.feature.account.setup.domain.entity
 import kotlinx.collections.immutable.toImmutableList
 
 enum class AuthenticationType {
-    PLAIN,
-    CRAM_MD5,
-    EXTERNAL,
-    OAUTH2,
+    PasswordCleartext,
+    PasswordEncrypted,
+    OAuth2,
     ;
 
     companion object {
-        val DEFAULT = PLAIN
+        val DEFAULT = PasswordEncrypted
```

Comment:
`PasswordCleartext` is available more often than `PasswordEncrypted`. So it should probably be the default.